src/index.jsx

```diff
@@ -53,10 +52,8 @@ function bootstrap() {
   return new Promise((resolve, reject) => {
     // We need to do this (in order) before bothering to render:
     // 1. Check status of the server
-    // 2. grab the configuration + load Google's auth2 library (in parallel)
```

Comment:
For the Google Sheets PR, I'm relying on loading the google client as a part of this call ([see spike PR](https://github.com/DataBiosphere/jade-data-repo-ui/pull/1114/files#diff-fd91852876d8943d1f7adbb51e497d186783e17321432d8b886bdf51b31fdf38R12)). Not sure how that will translate.

Reply:
Ok cool.  I think we can maybe add some branching logic to only show the sheet thing when we log in with google?  I wonder if maybe there's a way to use the api with the access token (provided we get the right scopes from B2C)